config/manifests/kustomization.yaml

```diff
@@ -38,4 +38,10 @@ patchesJson6902:
     version: v1alpha1
     kind: ClusterServiceVersion
     name: gitops-operator.v0.0.3
-  path: patches/description.yaml
\ No newline at end of file
+  path: patches/description.yaml
+- target:
```

Comment:
Since the annotation was manually added, can we directly remove it from the base CSV?

Reply:
Hi @chetan-rns , This will help us in keeping a track of the changes that we made to base CSV. Do you see any downside of having the entry in kustomization for this ?

Reply:
Hi @iam-veeramalla, the annotation was directly added to the base CSV which should have been done through kustomize. Since it's not part of the original base CSV generated by sdk do we still need to track it?

Reply:
Ack. Can you please take a look and approve